ui/src/containers/Header.js

```diff
@@ -12,9 +12,7 @@ class Header extends Component {
     const {user} = this.props;
     const helpDropdowns = [{
       text: "Documentation",
-      onSelect: () => {
-        console.log("Implement me!!");
-      }
+      href: 'http://docs.aerogear.org/'
```

Comment:
Should we add a comment here that once we've got documentation for MDC, the link should navigate you straight to MDC service?

Reply:
Makes sense, will add it.